config/config.h.in

```diff
@@ -57,6 +57,8 @@
 #cmakedefine USE_FNAME_CASE
 #define USEMAN_S 1
 
+#cmakedefine USE_CRNL
```

Comment:
Won't this make `#ifdef USE_CRNL` true for all platforms? I `win_defs.h` was probably an ok place for this.